program/dashboard/Dashboard_SpreadSheetFriendly.md

```diff
@@ -0,0 +1,72 @@
+# OpenHW Project Dashboard																												
+																												
+Last Updated October 4 2022																												
+																												
+## Status Key																												
+**PC = Project Concept** 																												
+&nbsp; &#8595;  																												
+**PL = Project Launch**  																												
+&nbsp; &#8595;  																												
+**PA = Plan Approved**																												
+&nbsp; &#8595;  																												
+**PF = Project Freeze**																												
+																												
+				 +																								

```

Comment:
The "+" character is extraneous.  The Markdown rendering on GitHub doesn't know how to handle it.  Recommend deleting this line altogether.